kolibri/core/assets/src/api-resource.js

```diff
@@ -187,8 +190,12 @@ class Model {
     return promise;
   }
 
+  get orderedUrlParams() {
+    return this.resource.urlParams.map((key) => this.urlParams[key]);
+  }
```

Comment:
calling `.map` on an object?

also, if `orderedUrlParams` is only used internally, necessary to make a new public computed attribute?